src/run_sign.py

```diff
@@ -23,7 +23,9 @@ def main():
     parser.add_argument("target", type=Path, help="Path to local manifest file or artifact directory.")
     parser.add_argument("--component", nargs="?", help="Component name")
     parser.add_argument("--type", nargs="?", help="Artifact type")
-    parser.add_argument("--sigtype", choices=ACCEPTED_SIGNATURE_FILE_TYPES, help="Type of Signature file", default=".asc")
+    parser.add_argument("--signatureType", choices=ACCEPTED_SIGNATURE_FILE_TYPES, help="Type of Signature file", default=".asc")
+    # This distributionPlatform argument is added for future use to extend the functionality of this library.
+    parser.add_argument("--distributionPlatform", nargs="?", help="The distribution platform")
```

Comment:
To be consistent with other tools i think this should just be `platform`. I would default it to `linux` and remove the comment above.

Reply:
Removed the comments and add the default value. I will leave the argument name to be `distributionPlatform` since that's what it used in some jenkinsfiles. e.g. [sign-standalone-artifacts.jenkinsfile](https://github.com/opensearch-project/opensearch-build/blob/0a20ec18c3a39770e870b2dade3c23b147962651/jenkins/sign-artifacts/sign-standalone-artifacts.jenkinsfile#L47https://github.com/opensearch-project/opensearch-build/blob/0a20ec18c3a39770e870b2dade3c23b147962651/jenkins/sign-artifacts/sign-standalone-artifacts.jenkinsfile#L47); [distribution-artifacts.jenkinsfile](https://github.com/opensearch-project/opensearch-build/blob/0a20ec18c3a39770e870b2dade3c23b147962651/jenkins/data-prepper/distribution-artifacts.jenkinsfile#L34)